client/checkout/utils/platform-checkout.js

```diff
@@ -1,3 +1,10 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { debounce } from 'lodash';
```

Comment:
Somehow I missed pushing my last commit so it didn't have all the changes needed. I updated the [webpack config](https://github.com/Automattic/woocommerce-payments/pull/3736/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR116) to bundle lodash instead of using it externally. This should only copy the debounce function into the js asset. I look at implementing our own debouce but it's not a trivial function so I decided not to.